app/views/microposts/_micropost.html.erb

```diff
@@ -11,5 +11,16 @@
 			<%= link_to "delete", micropost, method: :delete,
 											 data: { confirm: "Вы уверены?" } %>
 		<% end %>
+		<!-- I know it wery ugly and this code should be in model or in helper (I guess) -->
+		<% arr = micropost.votes.collect {|i| i[:grades].to_f } %>
```

Comment:
This right here triggers a database request each time you render a partial. 
This is really bad for performance reasons. 

You need to [read up on loading associations](https://guides.rubyonrails.org/active_record_querying.html#eager-loading-associations) and fix this problem.